### DISCUSSION TITLE
Form submission focus and scroll

### DISCUSSION THREAD

**Discussion by Tofandel** [quasarframework/quasar#13837] [Ideas / Proposals]
Currently when we submit a form with an error, it by defaults focuses the first invalid element, but it doesn't scroll it into view meaning the user may not see the error if the form is long and wonder why the form is not submitting

My workaround is `@validation-error="$event.$el.scrollIntoView({behavior: 'smooth', block: 'end', inline: 'nearest'})"`

But I'm wondering if this would have it's place in the core of quasar as it's a much better UX

**Comment by pdanpdan** [comment]
https://github.com/quasarframework/quasar/issues/13295
